app/actions/get_info.py

```diff
@@ -0,0 +1,16 @@
+from app.add_some_abilities.add_protection import protection
+from app.add_some_abilities.add_power import power
+from app.add_some_abilities.add_potion_abilities import add_ability_from_potion
+
+
+def get_knights_info(knightscfg: dict) -> list:
+    knights = []
+    for knight_name in knightscfg:
+        knights.append(knightscfg[knight_name])

```

Comment:
Consider using a more descriptive variable name than 'knight_name' to improve code readability, such as 'knight_key' or 'knight_id', since it represents a key in the dictionary.